self-hosted/migration/schema-then-data.md

```diff
@@ -105,46 +105,17 @@ owners, and settings. This doesn't include Timescale-specific schemas.
 
 </Procedure>
 
-### Troubleshooting
-
-If you see any of these errors during the migration process, you can safely
-ignore them. The migration still occurs successfully.
-
-`pg_restore` tries to apply the TimescaleDB extension when it copies your
-schema. This can cause a permissions error. Because TimescaleDB is already
-installed by default on Timescale Cloud, you can safely ignore this.
-
-```bash
-pg_restore: creating EXTENSION "timescaledb"
-pg_restore: creating COMMENT "EXTENSION timescaledb"
-pg_restore: while PROCESSING TOC:
-pg_restore: from TOC entry 6239; 0 0 COMMENT EXTENSION timescaledb
-pg_restore: error: could not execute query: ERROR:  must be owner of extension timescaledb
-```
-
-If you have continuous aggregates, you might get the following error. Ignore
-this, because you [restore your aggregates](#recreate-continuous-aggregates)
-later on.
-
-```bash
-pg_restore: error: could not execute query: ERROR:  relation "_timescaledb_internal._materialized_hypertable_x" does not exist
-```
-
-```bash
-pg_restore: WARNING:  no privileges were granted for "<..>"
-```
-
-## Restore hypertables in Timescale Cloud
+## Restore hypertables in Timescale
 
 After pre-data migration, your hypertables from your source database become
-regular PostgreSQL tables in Cloud. Recreate your hypertables in Cloud to
+regular PostgreSQL tables in Timescale. Recreate your hypertables in Timescale to
 restore them.
 
 <Procedure>
 
-### Restoring hypertables in Timescale Cloud
+### Restoring hypertables in Timescale
 
-1.  Connect to your Timescale Cloud database:
+1.  Connect to your Timescale database:
 
     ```sql
     psql "postgres://tsdbadmin:<CLOUD_PASSWORD>@<CLOUD_HOST>:<CLOUD_PORT>/tsdb?sslmode=require"
```

Comment:
This script also references cloud (but is in self-hosted)

Reply:
```suggestion
    psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
```